lib/Utils.php

```diff
@@ -300,6 +300,31 @@ public static function execCommandPipe($command, $args = null, $input = null, $d
         return $output;
     }
     
+    /**
+     * Fix contexts for https and missing trailing /
+     *
+     * @param array $object
+     * @return array correct object with http schema and @vocab.
+     */
+    public static function fixContexts(array $object): array
+    {
+        foreach($object as $key => $value) {
+            if (is_array($value)) {
+                $object[$key] = self::fixContexts($value);
+            }            
+            if ($key == "@context") {
+                $object[$key] = str_replace("https://schema.org", "http://schema.org", $object[$key]);
+                // I found cases in WP where trailing slashes were missing so handle that as well.
+                $object[$key] = $object[$key] === "http://schema.org" ? "http://schema.org/" : $object[$key];
+                $object[$key] = [
+                    "@vocab" => $object[$key]
+                ];
+            }
+        }
+
+        return $object;
+    }
```

Comment:
This method seems very easily testable which is good.  It seems like it would be a good idea to add some tests that call this directly to make sure it's replacing things as expected.  Tests could just be in the schema app repo.  What do you think?